examples/ReadExample.h

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (C) 2019 HERE Europe B.V.
+ * Copyright (C) 2019-2020 HERE Europe B.V.
```

Comment:
The previous version with the single `Examples.h` header looked better. Would it make sense to merge this header back to the `Examples.h`?

Reply:
I have excluded cache example from android\ios build, that is why headers are split.

Reply:
I see. It makes sense then.